pkg/apis/ceph.rook.io/v1beta1/types.go

```diff
@@ -145,6 +145,9 @@ type PoolSpec struct {
 	// The root of the crush hierarchy utilized by the pool
 	CrushRoot string `json:"crushRoot"`
 
+	// The device class the OSD should set to (options are: hdd, ssd, or nvme )
```

Comment:
```suggestion
	// The device class the OSD should set to (options are: hdd, ssd, or nvme)
```

Reply:
Completely missed these extra spaces. It should be ok now.